src/nemos/glm.py

```diff
@@ -407,6 +441,19 @@ def score(
             If X structure doesn't match the params, and if X and y have different
             number of samples.
 
+        Examples
+        --------
+        >>> # example input
+        >>> import numpy as np
+        >>> X, y = np.random.normal(size=(10, 2)), np.random.uniform(size=10)
+
+        >>> import nemos as nmo
+        >>> model = nmo.glm.GLM()
+        >>> model = model.fit(X, y)
+
+        >>> # get model score
+        >>> log_likelihood_score = model.score(X, y)
```

Comment:
```suggestion
        >>> log_likelihood_score = model.score(X, y)
        
        >>> # get a pseudo-R2 score
        >>> pseudo_r2 = model.score(X, y, score_type="pseudo-r2-McFadden")
```

Reply:
I still don't think we need to show both here

Reply:
I disagree

Reply:
It is an important parameter and it's a few extra lines, let's keep it